### DISCUSSION TITLE
how to use CSSHelpers outside your project?

### DISCUSSION THREAD

**Discussion by AlonsoK28** [ng-matero/ng-matero#470] [Q&A] [answered]
I want to use [CSS Helpers](https://ng-matero.github.io/ng-matero/utilities/css-helpers) into my project, this is possible ?

![image](https://user-images.githubusercontent.com/37458700/212815571-9c01070f-c1b2-4556-8553-8470e8a9bdbe.png)

**Comment by nzbin** [comment, marked as answer]
Just copy the [helpers](https://github.com/ng-matero/ng-matero/tree/master/src/styles/helpers) to your project and import it.
```scss
@use 'helpers/api';
```
In fact, there has another css library [snack-helper](https://github.com/snack-ui/snack-helper) that you can use.
